api/src/main/resources/messages_fr.properties

```diff
@@ -237,12 +237,29 @@ isanteplusreports.number_of_patients_with_malaria_hospitalized=Nombre de patient
 isanteplusreports.number_of_cases_of_confirmed_malaria_for_pregnant_women=Nombre de cas de la malaria confirm�e pour les femmes enceintes
 
 isanteplusreports.pnls.label=Rapport mensuel ART
-isanteplusreports.pnls.result.department.label=Dep't
+isanteplusreports.pnls.result.department.label=D�partement
 isanteplusreports.pnls.result.clinic.label=Clinique
 isanteplusreports.pnls.result.Males.label=M
 isanteplusreports.pnls.result.FeMales.label=F
-isanteplusreports.pnls.result.Subtotal.label=Sous-total
+isanteplusreports.pnls.result.Subtotal.label=Total
 isanteplusreports.pnls.result.BreastFeeding.label=Allaitement maternel
+isanteplusreports.pnls.result.NewPatients.label=R�cemment enr�l�
+isanteplusreports.pnls.result.Referenced.label=previously enrolled and referees 
+isanteplusreports.pnls.result.died.label=D�c�d�s
+isanteplusreports.pnls.result.voluntary.label=Voluntary
+isanteplusreports.pnls.result.denial.label=Le d�ni
+isanteplusreports.pnls.result.medical.label=M�dicale
+isanteplusreports.pnls.result.refered.label=R�f�r�
+isanteplusreports.pnls.result.other.label=Autre
+isanteplusreports.pnls.result.msm.label=MSM
+isanteplusreports.pnls.result.sexP.label=Professionnels du sexe		
+isanteplusreports.pnls.result.trans.label=transgenres		
+isanteplusreports.pnls.result.captv.label=prisonni�res
+isanteplusreports.pnls.result.drug.label=Utilisateurs de drogues
+isanteplusreports.pnls.result.newCtx.label=Patients VIH+ sous ARV nouvellement mis sous  CTX
+isanteplusreports.pnls.result.activeCtx.label=Patients actifs sous ARV sous CTX
+isanteplusreports.pnls.result.newTb.label=Nouveaux enrol�s sous ARV
+isanteplusreports.pnls.result.alreadyTb.label=Patients d�j� sous ARV 
 isanteplusreports.pnls.result.1-0.label=0-1 Yrs
```

Comment:
Please change the French label for _Yrs_ to _ans_ as in the excel template

Reply:
done